test/middleware/companyAuth.middleware.test.ts

```diff
@@ -0,0 +1,94 @@
+import 'reflect-metadata'
+
+import ApplicationLogger from 'ch-logging/lib/ApplicationLogger'
+import { SessionKey } from 'ch-node-session-handler/lib/session/keys/SessionKey'
+import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionInterfaces'
+import { assert } from 'chai'
+import { RequestHandler, Response } from 'express'
+import sinon from 'sinon'
+import { mock } from 'ts-mockito'
+
+import CompanyAuthMiddleware, { AuthConfig } from 'app/middleware/companyAuth.middleware'
+import DissolutionSession from 'app/models/dissolutionSession'
+import { JwtEncryptionService } from 'app/services/encryption/jwtEncryption.service'
+
+import { generateRequest } from 'test/fixtures/http.fixtures'
+import { generateDissolutionSession } from 'test/fixtures/session.fixtures'
+
+describe('AuthMiddleware', () => {
+
+  let middleware: RequestHandler
+
+  let encryptionService: JwtEncryptionService
+  let logger: ApplicationLogger
+  let authConfig: AuthConfig
+
+  const COMPANY_NUMBER = '12345678'
+
+  beforeEach(() => {
+    encryptionService = mock(JwtEncryptionService)
+    logger = mock(ApplicationLogger)
+
+    authConfig = {
+      chsUrl: '',
```

Comment:
seems like we should add some default values here so that we can assert parts of the redirectUrl